### ISSUE TITLE
SkyrimSE.exe+7BED0F

### ISSUE THREAD

**Issue by fenix31415** [Fikthenig/Crash-Bonanza#5]
Possible objects:
* EnchantmentItem
* BGSEquipSlot
* TESNPC
* Character
* ScrollItem or TESObjectWEAP or (smth with 'Spell', probably)

When occurs:
* Spell/scroll/staff and it's effects has inconsistent Target Type.
* This spell/scroll/staff in npc's inventory
* This npc entered combat

Reason: when npc want to choose item, it checks the broken one. It tries to find "most powerful effect", but finds nullptr. And then...

Fix: find broken item in "possible objects" section, set proper Target type (look at effects, use brain and logic, etc) or just complain to author, pointing to the reason and suggest the fix.

Appearance Kai - Mysticism - Path of Sorcery Patch. Latest version already fixed.

**Comment by Fikthenig** [comment]
SkyrimSE.exe+7BED0F
While the possible objects change with whatever is broken, is the main complaint "SkyrimSE.exe+7BED0F" always the same?

**Comment by fenix31415** [comment]
Yes. Objects Character, NPC, BGSEquipSlot and EnchantmentItem should also be same.

**Comment by Fikthenig** [comment]
That's a nice insight. I will leave this open for when I am updating the guide.
